<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Valgrind::Massif::View</class>
 <widget class="QWidget" name="Valgrind::Massif::View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>393</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="snapshotsLayout">
       <item>
        <widget class="QLabel" name="snapshotsLabel">
         <property name="text">
          <string>Snapshots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="snapshotsView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="treesLayout">
         <item>
          <widget class="QLabel" name="treesLabel">
           <property name="text">
            <string>Heap trees</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="treesView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="launchVisualizerButton">
           <property name="text">
            <string>Launch visualizer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>snapshotsView</tabstop>
  <tabstop>treesView</tabstop>
  <tabstop>launchVisualizerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
